codec/src/main/java/io/netty/handler/codec/compression/LzfEncoder.java

```diff
@@ -131,10 +133,10 @@ public LzfEncoder(boolean safeInstance, int totalLength, int compressThreshold)
                     " (expected: " + MIN_BLOCK_TO_COMPRESS + '-' + MAX_CHUNK_LEN + ')');
         }
 
-        if (compressThreshold >= 0 && compressThreshold < MIN_BLOCK_TO_COMPRESS) {
+        if (compressThreshold > 0 && compressThreshold < MIN_BLOCK_TO_COMPRESS) {
```

Comment:
I think we should not use `-1` as a special number here and use `MIN_BLOCK_TO_COMPRESS ` as our default.

Reply:
Fixed.